### ISSUE TITLE
Remove peerDependencies

### ISSUE THREAD

**Issue by fwh1990** [daybrush/moveable#410]
## Environments
* Framework name: react
* Framework version: 16.14
* Moveable Component version: react-moveable@0.26.1
* Testable Address(optional): 

## Description
Yarn always report warning messages:
```
warning " > react-moveable@0.26.1" has unmet peer dependency "framework-utils@>=0.3.4".
warning " > react-moveable@0.26.1" has unmet peer dependency "@scena/dragscroll@>=0.3.1".
```

To resolve it, you may remove peerDependencies from package.json since you have installed them by yourself in `dependencies`
```diff
{
    "dependencies": {
        "@daybrush/utils": "^1.4.0",
        "@egjs/agent": "^2.2.1",
        "@egjs/children-differ": "^1.0.1",
        "@scena/dragscroll": "^0.3.1",
        "@scena/matrix": "^1.1.1",
        "css-to-mat": "^1.0.2",
        "framework-utils": "^0.3.4",
        "gesto": "^1.2.1",
        "overlap-area": "^1.0.0",
        "react-css-styled": "^1.0.1"
    },
-   "peerDependencies": {
-       "framework-utils": ">=0.3.4",
-       "@scena/dragscroll": ">=0.3.1"
-   }
}
```

**Comment by daybrush** [comment]
@fwh1990 

okay I'll check it.!

**Comment by fwh1990** [comment]
Fixed by commit 4e34f25a7c7ecd39c4c5dd1a207af3b59ba6554b
